client/src/Try/Container.purs

```diff
@@ -85,7 +86,9 @@ _editor = SProxy
 loader :: Loader
 loader = makeLoader Config.loaderUrl
 
-foreign import setupIFrame :: EffectFn1 (Object JS) Unit
+type LoadCb = Effect Unit
+type FailCb = Effect Unit
```

Comment:
I don't think these type aliases are needed, but :shrug:

Reply:
I'm OK with them in this case because it can be otherwise difficult to tell which is the 'load' callback and which is the 'fail' callback. In other projects I also use records to help, ie. `{ fail :: Effect Unit, load :: Effect Unit }`.